src/main/java/org/eclipse/yasson/internal/serializer/CharArrayDeserializer.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Oracle and/or its affiliates. All rights reserved.
```

Comment:
2019 here

Reply:
I guess this should be done for the `CharArraySerializer.class`, too.
Does the header needs some more changes but the copyright date?